app/src/main/java/com/javahelp/frontend/ClientRegistrationVm.java

```diff
@@ -0,0 +1,13 @@
+package com.javahelp.frontend;
+
+import android.app.Application;
+
+import androidx.annotation.NonNull;
+import androidx.lifecycle.AndroidViewModel;
+import androidx.lifecycle.ViewModel;
+
+public class ClientRegistrationVm extends AndroidViewModel{
+    public ClientRegistrationVm(@NonNull Application application) {
+        super(application);
+    }
+}
```

Comment:
Is this the part where the findview by id etc come in? Like the methods I used in today's pr.

Reply:
I think the methods for registration for the front end will come in